components/LoadingCard.js

```diff
@@ -2,10 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Card } from 'antd';
 
+// todo rename CardWithLoading or uselss component ?
```

Comment:
Not useless, still used by the pages. I think it's good to have a middle man component between our usage of it and the framework, so that we can easily add or remove stuff from it without having to refactor every file. What do you think?
But as you said, we should move it into the `Card`.

Reply:
Hey, I move the component inside Card file